index.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Hello World!</title>
+    <link rel="stylesheet" type="text/css" href="css/styles.css">
+</head>
+<body>
+    <p>Hello World!</p>
+</body>
+</html>
```

Comment:
#### Nitpick :warning:
You don’t have to forget about the use of indentations because it makes your code more understandable.